pages/badges/badge-single.jsx

```diff
@@ -234,16 +234,26 @@ var BadgePage = React.createClass({
     );
   },
 
+  getShareCodes: function() {
+    var url = encodeURIComponent(window.location.toString());
+    var msg = encodeURIComponent(`I earned the Mozilla ${this.state.badge.title} badge`);
```

Comment:
do we not want to link to the badge itself? As a user I don't want to share "mozilla's badge program", I want to show off what *my* stuff, in this case the badge I just earned =)